src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.cs

```diff
@@ -73,8 +73,7 @@ public static Matrix3x2 Identity
         /// <param name="column">The index of the column containing the element to get or set.</param>
         /// <returns>The element at [<paramref name="row" />][<paramref name="column" />].</returns>
         /// <exception cref="ArgumentOutOfRangeException">
-        /// <paramref name="row" /> was less than zero or greater than the number of rows, or
-        /// <paramref name="column" /> was less than zero or greater than the number of columns
+        /// <paramref name="row" /> was less than zero or greater than the number of rows, or <paramref name="column" /> was less than zero or greater than the number of columns.
```

Comment:
Usually we separate exception strings if they involve different params.
```suggestion
        /// <paramref name="row" /> was less than zero or greater than the number of rows.
        /// -or-
        /// <paramref name="column" /> was less than zero or greater than the number of columns.
```